### PR TITLE
Coreclr compatibility: add vNext project

### PR DESCRIPTION
**Pull request by pherbel**
PR for Port Orleans to coreclr #368

Add vNext project to help the coreclr compatibility development. You can use vNext project with VS 2015

### CODE REVIEW

File: src/Orleans/project.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
+﻿{
```
Comment (gabikliot) [inline src/Orleans/project.json]: what is that file for?

File: global.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (gabikliot) [inline global.json]: what is that file for?

File: src/Orleans/project.json (hunk 1 above)
Comment (pherbel) [inline src/Orleans/project.json]: project.json is real project file and it is portable . The xproj is not needed and you can develop a coreclr project without Visual Studio.
Check this out:
http://dotnet.readthedocs.org/en/latest/dnx/overview.html
Also it defines the nugget package.

File: global.json (hunk 2 above)
Comment (pherbel) [inline global.json]: It is something like a solution file. It is help to resolve the package references on solution level.

**Comment by gabikliot** [comment]
Feels like Microsoft.Orleans.vNext.sln and global.json  maybe need to be inside src folder, next to Orleans.sln?
Will merge now, but might need to move those 2 files later on.

**Pull request merged**